js/components/app/buttons/ButtonVurdering.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Knapp from 'nav-frontend-knapper';
+
+const texts = {
+    buttonVurdering: 'Gi din vurdering',
+};
+
+const ButtonVurdering = (
+    {
+        click,
+        text = texts.buttonVurdering,
+    }) => {
+    return (
+        <Knapp
+            autoFocus
+            type="standard"
+            className="knapp--gi-vurdering"
+            onClick={click}>
+            {text}
+        </Knapp>
```

Comment:
Skal dette være en [miniknapp](https://design.nav.no/components/knapp#mini)?
Det kommer ikke klart frem av skissene, der er knappen stor, men teksten er ikke UPPERCASE.
Tilsvarende knapp i [oppfolgingsplanen](https://github.com/navikt/oppfolgingsplan/blob/b8217a8925cf266c8b70cd8bdc3c20a2542a474d/js/components/oppfolgingsplan/tiltak/TiltakKnapper.js#L54) er mini.

```suggestion
     <Knapp
            mini
            autoFocus
            onClick={click}>
            {text}
        </Knapp>
```